docker-compose.yml

```diff
@@ -1,5 +1,24 @@
 version: '3.1'
 services:
+  localstack:
+    image: localstack/localstack
+    env_file:
+      - .docker/local.env
+    ports:
+      - '4566:4566'
```

Comment:
Not for now, but a note at somepoint we should start changing our localstack ports so we can run multiple subgraphs.